service/matching/handler/engine.go

```diff
@@ -441,6 +466,16 @@ func (e *matchingEngineImpl) PollForDecisionTask(
 		tag.WorkflowTaskListName(taskListName),
 		tag.WorkflowDomainID(domainID),
 	)
+	event.Log(event.E{
+		TaskListName: taskListName,
+		TaskListKind: taskListKind,
+		TaskListType: persistence.TaskListTypeDecision,
+		TaskInfo: persistence.TaskInfo{
+			DomainID: domainID,
+		},
+		EventName: "Received PollForDecisionTask",
+		Host:      e.config.HostName,
```

Comment:
do we need to include `request.GetForwardedFrom()` in the payload?

Reply:
added.